app/assess/templates/components/answer_view_table.html

```diff
@@ -0,0 +1,23 @@
+{% macro render(question, question_search_endpoint) -%}
+<table class="govuk-table">
```

Comment:
nice macro for a table :) this is different from the default govuk table in some way to warrant its existence?

Reply:
only that its html and i prefer looping with the html than the nunjuck macro. The macros are not within the scope of this ticket but its a fair point - why reinvent.  So i will look into that 💯